_protected/app/system/modules/mail/forms/SearchMailForm.php

```diff
@@ -19,13 +19,35 @@ public static function display()
     {
         $oForm = new \PFBC\Form('form_search');
         $oForm->configure(['action' => self::getActionUrl() . PH7_SH, 'method' => 'get']);
-        $oForm->addElement(new Search(t('Search a message:'), 'looking', ['description' => t('Enter a keyword in the Subject, Contents, Author (username, first name, last name) or message ID.')]));
-        $oForm->addElement(new Select(t('Browse By:'), 'order', [SearchCoreModel::TITLE => t('Subject'), SearchCoreModel::USERNAME => t('Author (username)'), SearchCoreModel::SEND_DATE => t('Recent')]));
+        
+        $oForm->addElement(new \PFBC\Element\HTMLExternal('<div class="msg-search-bar">'));
+        
+        $oForm->addElement(new \PFBC\Element\HTMLExternal('<div class="msg-search-input">'));
+        $oForm->addElement(new \PFBC\Element\Search('', 'looking', ['placeholder' => t('Keyword'), 'style' => 'width:250px']));
+        $oForm->addElement(new \PFBC\Element\HTMLExternal('</div>'));
+
+        $oForm->addElement(new \PFBC\Element\HTMLExternal('<div class="msg-search-input">'));
+        $oForm->addElement(new Select('', 'order', [SearchCoreModel::TITLE => t('Subject'), SearchCoreModel::USERNAME => t('Author (username)'), SearchCoreModel::SEND_DATE => t('Recent')]));
+        $oForm->addElement(new \PFBC\Element\HTMLExternal('</div>'));
+
+        $oForm->addElement(new \PFBC\Element\HTMLExternal('<div class="msg-search-input">'));
         if (!self::isAdminLoggedAndNotUser()) {
-            $oForm->addElement(new Select(t('Where:'), 'where', [MailModel::INBOX => t('Inbox'), MailModel::OUTBOX => t('Outbox'), MailModel::TRASH => t('Trash')]));
+            $oForm->addElement(new Select((''), 'where', [MailModel::INBOX => t('Inbox'), MailModel::OUTBOX => t('Outbox'), MailModel::TRASH => t('Trash')]));
```

Comment:
Was it on purpose that you omitted the `t('Where:')` string?

Reply:
This where: is being echo'd in quotes because its being used for the label of the form element. The 'where' which is the second parameter is the MySQL clause. Have a look in a browser you will understand what I mean.